test/integration/tas/tas_test.go

```diff
@@ -261,10 +261,7 @@ var _ = ginkgo.Describe("Topology Aware Scheduling", ginkgo.Ordered, func() {
 						Ready().
```

Comment:
Please drop in this file the not-ready tolerations like these: https://github.com/kubernetes-sigs/kueue/blob/8cc315716e31d8439665452f8e5627136e352363/test/integration/tas/tas_test.go#L635-L638

Reply:
Done